docs/docusaurus/docs/reference/learn/data_quality_use_cases/integrity.md

```diff
@@ -0,0 +1,278 @@
+
+Validating the integrity of this financial data involves checking the consistency and accuracy of relationships between the `transactions` and `transfer_balances` tables, as well as verifying that business rules and constraints are met.
+
+## Key distribution Expectations
+
+Great Expectations provides several Expectations designed for validating relationships between data elements within a single table. However, if you want to validate data relationships across multiple tables, the current built-in Data Integrity Expectations won't work out of the box. To overcome this limitation, you have two options:
+
+1. Create a view on the database that combines the tables you want to validate, and then use the built-in Expectations on that view.
+2. Create custom SQL Expectations to validate relationships directly in the database.
+
+In this section, we'll cover the first option by discussing built-in Expectations that can be applied to a combined view of multiple tables. Later, in the "Example: Validate cross-table balance integrity" section, we'll explore the second option, demonstrating how to create and use custom SQL Expectations for validating relationships across separate tables.
+
+:::TODO:::
+
+![Add an integrity Expectation in GX Cloud](integrity_resources/integrity_add_expectation.gif)
+
+### Expect Column Pair Values To Be Equal
+
+Validates that values in one column match corresponding values in another column.
+
+**Use Case**: Verify that the reference number and confirmation code match for each transaction.
+
+```python
+gxe.ExpectColumnPairValuesToBeEqual(
+    column_A="reference_number",
+    column_B="confirmation_code"
+)
+```
+
+<small>View `ExpectColumnPairValuesToBeEqual` in the [Expectation Gallery](https://greatexpectations.io/expectations/expect_column_pair_values_to_be_equal).</small>
+
+
+### Expect Multicolumn Sum To Equal
+
+Ensures that the sum of multiple columns equals an expected value, useful for validating financial calculations and balance checks.
+
+**Use Case**: Validate sum-to-zero checks in the `transfer_balances` table.
+
+```python
+gxe.ExpectMulticolumnSumToEqual(
+    column_list=["adjustment", "sender_debit", "recipient_credit"],
+    sum_total=0.0
+)
+```
+
+<small>View `ExpectMulticolumnSumToEqual` in the [Expectation Gallery](https://greatexpectations.io/expectations/expect_multicolumn_sum_to_equal).</small>
+
+
+### Expect Column Pair Values A To Be Greater Than B
+
+Verifies that values in one column are consistently greater than related values in another column.
+
+**Use Case**: Ensure transfer amounts are always greater than or equal to adjustments.
```

Comment:
My knowledge of banking is only based on my experience as a consumer - would an adjustment ever be larger than a transfer amount in the real world? For my own edification for working on this article, what are common reasons to have an adjustment on a transfer?

Reply:
One example that I can think of: if the transfer amount is tiny and there's some fee or whatever, then yes, adjustment could be greater than the transfer amount.

Some reasons to have an adjustment on a transfer:
1. backdated transactions
2. fee
3. exchange rate adjustment
4. customer service; e.g. goodwill credit, dispute resolution